meercat/dashboard/views.py

```diff
@@ -1583,15 +1583,16 @@ def handle_flash(proj_object, filenames, project_info):
     proj_name = proj_object.name
     file_lines = []
     for filename in filenames:
-        name, extension = os.path.splitext(filename)
-        if (extension and extension in project_info['extensions']) or (not extension and filename in project_info['filenames']):
-            with open(str(settings.REPOS_DIR)+'/'+proj_name+'/'+filename, 'r') as f:
-                lines = f.readlines()
-                f.close()
-            file_lines.append((filename, name,  extension, lines))
-        else:
-            print(f'Uncheckable currently: {filename}')
-            file_lines.append((filename, name,  extension, None))
+        if os.path.isfile(str(settings.REPOS_DIR)+'/'+proj_name+'/'+filename):
+            name, extension = os.path.splitext(filename)
+            if (extension and extension in project_info['extensions']) or (not extension and filename in project_info['filenames']):
+                with open(str(settings.REPOS_DIR)+'/'+proj_name+'/'+filename, 'r') as f:
```

Comment:
Maybe also check for file existence (to avoid an ugly python exception)?

Reply:
os.path.isfile() will return false if file doesn't exist.

Reply:
Cool!